README.md

```diff
@@ -34,11 +38,16 @@ You will need thge following dependencies installed:
 The following commands worked for me on my Mac.
 No other guarantees are offered.
 
-    git clone git@github.com:holdenweb/sheetpub.git
-    cd sheetpub
-    poetry env use 3.9      # Includes support for 3.7 through 3.10
-    # Before continuinng, set up your environment (PYTHONPATH= ...)
-    # Ensure that the `src\transformers` directory is on there!
+    git clone git@github.com:holdenweb/python-training.git
+    cd python-training
+    poetry env use 3.9      # Includes support for 3.7 through 3.10    
```

Comment:
Should we add `poetry shell` before the install command? So that dependencies are only installed in the virtual environment ?

Reply:
No need. If poetry sees no current virtualenv it will activate the projects current environment (as determined by `poetry env use X.Y`).